docs/content/custom-replication-system.md

```diff
@@ -119,4 +139,9 @@ public class CubeColorSendSystem : CustomSpatialOSSendSystem<SpatialOSCubeColor>
 ```
 
 ----
-**Give us feedback:** We want your feedback on the Unity GDK and its documentation  - see [How to give us feedback](../../README.md#give-us-feedback).
\ No newline at end of file
+**Give us feedback:** We want your feedback on the Unity GDK and its documentation  - see [How to give us feedback](../../README.md#give-us-feedback).
+
+<!-- 
+Questions to deal with (but not limited to):
```

Comment:
@zeroZshadow, please respond to @bethaitman's question here. Specifically:
Does the `SpatialOSSendSystem` check the `DirtyBit` after it's been set to true?
Is that what triggers the `SpatialOSSendSystem` check?

thanks

Reply:
`SpatialOSSendSystem` is run at the end of a frame, after all the custom send systems have run.
It will go over all Spatial Components, checks if the DirtyBit is set to true, and will send an update for those that have it.
It will change this back to false after doing so.
Custom send systems will manually set the dirty bit to false.